.travis.yml

```diff
@@ -1,8 +1,18 @@
-language: node_js
 sudo: true
+
+language: node_js
+
 node_js:
-  - "0.10"
-  - "0.12"
-  - "iojs"
-  - "4"
-  - "5"
+  - 6
+  - 5
+  - 4
+  - 0.12
+  - 0.10
+  - iojs
+
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm install -g npm@latest
```

Comment:
similarly, we *want* to test the stock npm version, and not blindly update to latest. `tape` needs to work when installed with npm 2 as well as npm 3.